packages/dds/tree/src/feature-libraries/object-forest/objectForest.ts

```diff
@@ -391,14 +378,15 @@ class Cursor implements ITreeSubscriptionCursor {
         this.keyStack.pop();
         // TODO: maybe compute siblings lazily or store in stack? Store instead of keyStack?
         this.siblings = this.parentStack.length === 0 ?
-            // eslint-disable-next-line @typescript-eslint/no-non-null-assertion
+        // eslint-disable-next-line @typescript-eslint/no-non-null-assertion
             this.forest.getRoot(this.root!) :
-            this.parentStack[this.parentStack.length - 1].fields.get(this.keyStack[this.keyStack.length - 1]);
+            (this.parentStack[this.parentStack.length - 1].fields ?? {}
+                )[this.keyStack[this.keyStack.length - 1] as string];
```

Comment:
In my opinion this is getting to be a bit difficult to read, but I don't have a strong opinion about to change it.

Reply:
I'll see if I can fix this up in my next batch of changes. It is pretty confusing